packages/assets-controllers/src/token-prices-service/codefi-v2.test.ts

```diff
@@ -829,8 +1055,9 @@ describe('CodefiTokenPricesServiceV2', () => {
         clock.restore();
       });
 
-      it('does not call onDegraded when requests succeeds faster than threshold', async () => {
```

Comment:
We don't need these tests because this functionality is already covered in the tests for `createServicePolicy`. We just need to test that the `onDegraded` callback option works somehow.